examples/README.md

```diff
@@ -0,0 +1,75 @@
+
+### 2- Prediction of HER2 status in breast cancer from H&E stained whole slide images
+This example notebook demonstrates how the functionalities available in TIAToolbox can be used to reproduce the ["SlideGraph+ method" ("SlideGraph+: Whole Slide Image-Level Graphs to Predict HER2Status in Breast Cancer" by Lu et al. (2021))](https://arxiv.org/abs/2110.06042) to predict HER2 status of breast cancer samples from H&E stained whole slide images. As a brief overview, this method involves several steps to generating a graph that represents a whole slide image (WSI) and then directly feeding it into a special convolutional graph network, called SlideGraph, to predict whether that WSI is HER2 negative or positive.
+
+![image](https://user-images.githubusercontent.com/74412979/145244421-ad2f28fe-1361-44b8-a82f-707fd72b0a28.png)
+
+
+## B) Examples of TIAToolbox functionalities
+List of Jupyter notebooks that are dedicated to explaining the essential functionalities of TIAToolbox.
+
+### 1- Reading Whole Slide Images ([01-wsi-reading.ipynb](https://github.com/TissueImageAnalytics/tiatoolbox/blob/master/examples/01-wsi-reading.ipynb))
+This notebook shows you how you can use TIAToolbox to read different kinds of WSIs. TIAToolbox supports various WSI formats acquired by different sources a list of which is available [here](https://tia-toolbox.readthedocs.io/en/latest/usage.html?highlight=wsiread#tiatoolbox.wsicore.wsireader.get_wsireader). In this example, you will also learn about a couple of well-known techniques for WSI mask and patch extraction.
+
+![image](https://user-images.githubusercontent.com/74412979/145223963-f5cc3efc-5762-43c1-b040-c1f738a98e1b.png) ![image](https://user-images.githubusercontent.com/74412979/145224002-b61eb074-5b55-45c9-a45c-9b527437be2c.png)
+
+## 2- Stain normalisation of histology images ([02-stain-normalisation.ipynb](https://github.com/TissueImageAnalytics/tiatoolbox/blob/master/examples/02-stain-normalisation.ipynb))
+Stain normalisation is a common pre-processing step in computational pathology, whose objective is to reduce the colour variation in histology images (caused by using scanners or different staining protocols) to a minimum that has no significant impact on clinical/computational workflow. It has been shown in many studies that stain normalisation can make the algorithm more robust against the domain shift problem seen in histology images taken under different circumstances. TIAToolbox makes a few different stain-normalisation algorithms available to the user, and we demonstrate how to use them in this notebook. The implemented stain normalisation methods in TIAToolbox are:
+- Reinhard stain normalisation
+- Ruifork 
+- Macenko
+- Vahadane
+
+but you can use a custom stain matrix for stain normalisation as well.
+
+![image](https://user-images.githubusercontent.com/74412979/145226029-0cdcf94b-eb65-46ba-8f35-94bb6c457fab.png)
```

Comment:
same with this image. the text is not good in dark mode.

Reply:
corrected in commit eebdd8b